### PR TITLE
Completed Design -1 HashSet and minStack- Please Review my design-1 Submission

### PR DESCRIPTION
**Pull request by Butulparveen**

### CODE REVIEW

**Comment by Butulparveen** [comment]
Design 1 and 2 completed please check my submission

File: MinStack.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+//Microsfot interview question 
+//Time and Space complexity = O(1)
```
Comment (daredevilteja) [inline MinStack.java:2]: space complexity should be O(n). Since you are using a stack of dynamic length

**Review by daredevilteja — COMMENTED** [review]
Good Work! Keep it up

**Comment by daredevilteja** [comment]
Good Work! Keep it up

**Comment by daredevilteja** [comment]
Good work! Keep it up

**Pull request closed without merging**